VocaDbModel/Service/OtherService.cs

```diff
@@ -98,20 +93,21 @@ class EntryComparer : IEqualityComparer<IEntryWithNames> {
 				.OrderByDescending(a => a.RatingAverageInt)
 				.ToArray();
 
-			var popularAlbumsCached = random
-				.Select(a => new TranslatedAlbumContract(a))

```

Comment:
Why was `TranslatedAlbumContract` used here?

Reply:
From the log I can see that languagePreference was added to cache key much later. Originally I intended the same result to be cached for all languages. It could be a mistake that languagePreference was added to cache key. It might be unnecessary. 

It's not a huge issue since it won't cause any invalid results, just that the result is cached for each language separately. And of course using TranslatedAlbumContract is unnecessary if caching per language.

Reply:
Should I remove the `TranslatedAlbumContract` class?